tests/e2e/linalg_ext_ops/attention.mlir

```diff
@@ -12,3 +12,101 @@ func.func @attention() {
   ) : tensor<1x4x4xf32>
   return
 }
+
+func.func @attention1() {
```

Comment:
I don't have a good suggestion, but I wonder if we can make function names more meaningful?

Reply:
Updated, thanks!